Doc/howto/argparse.rst

```diff
@@ -841,6 +841,75 @@ translated messages.
 
 To translate your own strings in the :mod:`argparse` output, use :mod:`gettext`.
 
+Custom type converters
+======================
+
+The :mod:`argparse` module allows you to specify custom type converters for
+your command-line arguments. This allows you to modify user input before it's
+stored in the :class:`argparse.Namespace`. This can be useful when you need to
+pre-process the input before it is used in your program.
+
+For example, let's say that you want to accept a string that represents a
+fraction, and you want to convert it to a :class:`fractions.Fraction` object.
+
+Here is how you can do it::
+
+   import argparse
+   import fractions
+
+   parser = argparse.ArgumentParser()
+   parser.add_argument("x", type=lambda s: fractions.Fraction(s))
```

Comment:
This is just `type=fractions.Fraction`. This is not different from `type=int`, `type=float`, etc, shown in the module documentation and in the examples above. In general, I do not think that the first example is useful, as it is too trivial. Examples above imply that you already know how to use `type` in such simple case.